.travis.yml

```diff
@@ -53,6 +59,9 @@ matrix:
             - ccache
             - libdbus-glib-1-dev
             - libedit-dev
+            - libglib2.0-dev
+            - libxml2-dev
+            - libgtk-3-dev
```

Comment:
You don't need libxml2 and gtk in your case, that's only needed for some of the (gtk-based) example applications in the GStreamer bindings repo :)